tools/projmgr/src/ProjMgrYamlEmitter.cpp

```diff
@@ -895,20 +897,21 @@ bool ProjMgrYamlBase::WriteFile(YAML::Node& rootNode, const std::string& filenam
 }
 
 bool ProjMgrYamlEmitter::GenerateCbuildIndex(ProjMgrParser& parser,
-  const vector<ContextItem*> contexts, const string& outputDir) {
+  const vector<ContextItem*> contexts, const string& outputDir,
```

Comment:
Same question here. Passing by reference to avoid copying the vector? Or remove the `const` to express copy is intended.

```suggestion
  const vector<ContextItem*>& contexts, const string& outputDir,
```